packages/components/src/components/vertical-tab/vertical-tab-header.tsx

```diff
@@ -1,11 +1,46 @@
-import classNames from 'classnames';
 import React from 'react';
+import classNames from 'classnames';
 import { NavLink } from 'react-router-dom';
 import { getKebabCase } from '@deriv/shared';
 import Counter from '../counter';
 import Icon from '../icon/icon';
 
-const HeaderIcon = ({ icon, is_active }) => (
+export type THeaderIcon = {
+    icon: string;
+    is_active: boolean;
+};
+
+export type THeader = {
+    text?: string;
+    path?: string;
+};
+
+export type TItem = {
+    component?: typeof React.Component;
+    count?: number;
+    default?: boolean;
+    getTitle?: () => string;
```

Comment:
```suggestion
```
@yauheni-deriv Please try extending TRoute from shared instead of writing all properties all over again

Reply:
is it a good practice to extend type from other packages? it is not even exported.
as I see, cashier package also has its own routes types.
I believe we should use types within one package

Reply:
@yauheni-deriv i can see that we have TRoute type duplicated in different packages. It's a good practice to use shared package for something that is shared across packages.🙂  This is the main idea of shared package. Please export existing TRoute from shared and extend it here.

Reply:
Leaving it as it is after some checks and discussions